etree.go

```diff
@@ -135,6 +135,20 @@ func NewDocument() *Document {
 	}
 }
 
+// GetPath returns the path of the element corresponding to the parent element or document root
+func (e *Element) GetPath(parent *Element) string {
+	parts := []string{}
+
+	for element := e; element != nil && element.Tag != "" && element != parent; element = element.Parent() {
+		parts = append(parts, "")
+		copy(parts[1:], parts)
+		parts[0] = element.Tag
+	}
+
+	path := strings.Join(parts, "/")
+	return "./" + path
+}
+
```

Comment:
Maybe call this `GetRelativePath` and add another method called `GetPath` (without a parameter) that returns the absolute path of the element.

Also, for the parameter, I'd probably rename `parent` to `source`, since that is the source node you're seeking a relative path from.

Reply:
👍 I've made the changes requested.